doc/nrf/releases/release-notes-latest.rst

```diff
@@ -39,7 +39,8 @@ nRF9160
   * :ref:`lib_nrf_cloud` library:
 
     * Added cell-based location support to :ref:`lib_nrf_cloud_agps`.
-    * Added Kconfig option :option:`CONFIG_NRF_CLOUD_AGPS_SINGLE_CELL_ONLY` to obtain cell-based location from nRF Connect for Cloud instead of using the modem's GPS.
+    * Added Kconfig option :option:`CONFIG_NRF_CLOUD_CELL_POS` to obtain cell-based location from nRF Connect for Cloud instead of using the modem's GPS.
+    * Added Kconfig option :option:`CONFIG_NRF_CLOUD_CELL_POS_SINGLE_CELL_ONLY` to obtain approximate location using a single cell tower.
```

Comment:
update note to say `CONFIG_NRF_CLOUD_CELL_POS_SINGLE_CELL_ONLY` has been removed and that single cell has been separated from GPS/AGPS.

Reply:
I guess this note is not required as this flag is added then removed after NCSv1.5.0 and before NCSv1.6.0, so just like not happened